README.md

```diff
@@ -186,63 +191,85 @@ for (Bucket bucket : storage.list().iterateAll()) {
 #### Complete source code
 
 In
-[CreateAndListBucketsAndBlobs.java](../../google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/CreateAndListBucketsAndBlobs.java)
+[CreateAndListBucketsAndBlobs.java](https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/snippets/CreateAndListBucketsAndBlobs.java)
 we put together examples creating and listing buckets and blobs into one program. The program assumes that you are
 running on Compute Engine or from your own desktop. To run the example on App Engine, simply move
 the code from the main method to your application's servlet class and change the print statements to
 display on your webpage.
 
-Troubleshooting
----------------
+### Example Applications
 
-To get help, follow the instructions in the [shared Troubleshooting document](https://github.com/googleapis/google-cloud-common/blob/master/troubleshooting/readme.md#troubleshooting).
-
-Transport
----------
-Storage uses HTTP for the transport layer.
-
-Java Versions
--------------
-
-Java 7 or above is required for using this client.
+- [`StorageExample`](https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-examples/src/main/java/com/google/cloud/examples/storage/StorageExample.java) is a simple command line interface that provides some of Cloud Storage's functionality.  Read more about using the application on the [`StorageExample` docs page](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-examples/README.md).
+- [`Bookshelf`](https://github.com/GoogleCloudPlatform/getting-started-java/tree/master/bookshelf) - An App Engine application that manages a virtual bookshelf.
+  - This app uses `google-cloud` to interface with Cloud Datastore and Cloud Storage. It also uses Cloud SQL, another Google Cloud Platform service.
+- [`Flexible Environment/Storage example`](https://github.com/GoogleCloudPlatform/java-docs-samples/tree/master/flexible/cloudstorage) - An app that uploads files to a public Cloud Storage bucket on the App Engine Flexible Environment runtime.
 
-Testing
--------
 
-This library has tools to help make tests for code using Cloud Storage.
 
-See [TESTING] to read more about testing.
 
-Versioning
-----------
+## Troubleshooting
 
-This library follows [Semantic Versioning](http://semver.org/).
+To get help, follow the instructions in the [shared Troubleshooting document][troubleshooting].
 
-It is currently in major version one (``1.y.z``), which means that the public API should be considered stable.
+## Java Versions
 
-Contributing
-------------
+Java 7 or above is required for using this client.
 
-Contributions to this library are always welcome and highly encouraged.
+## Versioning
```

Comment:
@chingor13 how can I update versioning information for this library based on discussions around the Storage interface?

Reply:
I'll need to add an opt-out in the shared template configuration

Reply:
Gotcha, is this something you want to do after this PR?